vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -211,6 +243,12 @@ bool TextViewManager::UpdateProperty(
     if (react::uwp::IsValidColorValue(propertyValue)) {
       static_cast<TextShadowNode *>(nodeToUpdate)->m_ColorValue = react::uwp::ColorFrom(propertyValue);
     }
+  } else if (propertyName == "hitTestStrategy") {
+    if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::String) {
+      static_cast<TextShadowNode *>(nodeToUpdate)->useBlockHitTest = propertyValue.AsString() == "block";
```

Comment:
can you add a comment defining the 'block' strategy?

Reply:
@rectified95 I'll likely remove the block strategy for now.